statistics/build.gradle

```diff
@@ -41,10 +42,6 @@ dependencies {
 
     // Room
     implementation AndroidX.room.runtime
-    implementation AndroidX.room.rxJava2
-    kapt AndroidX.room.compiler
-    testImplementation AndroidX.room.testing
-    androidTestImplementation AndroidX.room.testing
```

Comment:
We have room entities here but we don't need to have the room annotation processor here. We just need access to the annotations. The app module will take care of the code generation.

This should also improve build times a bit